book/src/simple-payment-and-state-verification.md

```diff
@@ -52,61 +52,47 @@ A receipt is a minimal proof that; a transaction has been included in a block,
-
-An Entry-Merkle is a Merkle Root including all transactions in the entry, sorted
-by signature.
+A Block-Merkle is the Merkle Root of all the Entry-Merkles sequenced in the block.
 
 <img alt="Block Merkle Diagram" src="img/spv-block-merkle.svg" class="center"/>
 
-A Block-Merkle is a Merkle root of all the Entry-Merkles sequenced in the block.
+A Bank-Hash is the hash of the concatenation of the Block-Merkle and Accounts-Hash
+
+<img alt="Bank Hash Diagram" src="img/spv-bank-hash.svg" class="center"/>
+
+An Accounts-Hash is the hash of the concatentation of the state hashes of each
+account modified during the current slot.
+
 Transaction status is necessary for the receipt because the state receipt is
 constructed for the block. Two transactions over the same state can appear in
 the block, and therefore, there is no way to infer from just the state whether a
 transaction that is committed to the ledger has succeeded or failed in modifying
 the intended state. It may not be necessary to encode the full status code, but
 a single status bit to indicate the transaction's success.
 
-### State Merkle Path
-
-A state receipt provides a confirmation that a specific state is committed at the
-end of the block. Inter-block state transitions do not generate a receipt.
-
-For example:
-
-* A sends 5 Lamports to B
-* B spends 5 Lamports
-* C sends 5 Lamports to A
-
-At the end of the block, A and B are in the exact same starting state, and any
-state receipt would point to the same value for A or B.
+### Account State Verification
 
-The Bank-Merkle is computed from the Merkle Tree of the new state changes, along
-with the Previous Bank-Merkle, and the Block-Merkle.
+An account's state (balance or other data) can be verified by submitting a
+transaction with a ___TBDa_Instruction___ to the cluster. The client can then use
+a [Transaction Inclusion Proof](#transaction-inclusion-proof) to verify whether
+the cluster agrees that the account has reached the expected state.
 
-<img alt="Bank Merkle Diagram" src="img/spv-bank-merkle.svg" class="center"/>
-
-A state receipt contains only the state changes occurring in the block. A direct
-Merkle Path to the current Bank-Merkle guarantees the state value at that bank
-hash, but it cannot be used to generate a “current” receipt to the latest state
-if the state modification occurred in some previous block. There is no guarantee
-that the path provided by the validator is the latest one available out of all
-the previous Bank-Merkles.
-
-Clients that want to query the chain for a receipt of the "latest" state would
-need to create a transaction that would update the Merkle Path for that account,
-such as a credit of 0 Lamports.
+A complication of this method is avoiding bank accounts squashing discarding
+information required to verify blocks referenced by an inclusion proof. A solution
+is to instead snapshot/checkpoint these banks by submitting a ___TBDb_Instruction___,
```

Comment:
This is no longer necessary. The bank hash is derived from the snapshot image

Reply:
But we may want the bank hash to be the hash of the block transaction merkle root, hash of the snapshot image and the previous bank hash

Reply:
we don’t have an account inclusion proof. So you still need a tx.  Shove off :)